src/commands/Configuration/banpool.ts

```diff
@@ -0,0 +1,28 @@
+import Command from "../../structures/Command";
+import BulbBotClient from "../../structures/BulbBotClient";
+import create from "./banpool/create";
+import _delete from "./banpool/delete";
```

Comment:
Is it possible to combine this subcommand into the `!banpool remove` subcommand? I'd prefer to maintain all command names matching their file names. Required for load/unload commands

Reply:
I'm aware of the delete operator. Tbh I'm not really sure what my problem was, the filename is the important part not the import variable name, and it's already "delete", so seems like I was just wrong and there wasn't actually problem.